docs/develop/run.md

```diff
@@ -98,29 +224,67 @@ $ curl -k -XGET https://localhost:5000/api/records/ | jq .
 You can create a new record using the API:
 
 ```console
-$ curl -k -XPOST -H "Content-Type: application/json" https://localhost:5000/api/records/ -d '
-{
-    "access": {
-        "metadata_restricted": "false",
-        "files_restricted": "false"
-    },
-    "access_right": "open",
-    "contributors": [{"name": "Jon Doe"}, {"name": "Ian Matyssik"}],
-    "description": "Photo-analysis of doge in crystalline structures",
-    "owners": [1],
-    "publication_date": "05/11/2019",
-    "resource_type": {
-        "type": "image",
-        "subtype": "photo"
+$ curl -k -XPOST -H "Content-Type: application/json" https://localhost/api/records/ -d '{
+    "_access": {
+		"metadata_restricted": "false",
+		"files_restricted": "false"
+	},
+	"_owners": [1],
+	"_created_by": 1,
+	"access_right": "open",
+	"resource_type": {
+		"type": "image",
+		"subtype": "photo"
```

Comment:
Same note about indentation.

Reply:
damn this one was showing correctly on my pc hehe